quota/mysqlqm/mysql_quota.go

```diff
@@ -124,3 +130,29 @@ func countFromTable(ctx context.Context, db *sql.DB) (int, error) {
 	}
 	return count, nil
 }
+
+// turnOffInformationSchemaCache turn off statistics caching for MySQL 8
+// To always retrieve the latest statistics directly from the storage engine and bypass cached values, set information_schema_stats_expiry to 0.
+// See https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_information_schema_stats_expiry
+func turnOffInformationSchemaCache(ctx context.Context, db *sql.DB) error {
```

Comment:
Will this fail safely for all versions of MySQL prior to 8 or should we check the version and only execute on version 8+?

Reply:
Yes, it will fail safely for versions before MySQL 8.0. This is because this function uses `SHOW VARIABLES LIKE` statement. Executing this statement in versions before MySQL 8 is equivalent to searching a system variable that does not exist, and it will only return empty result rather than error.

Furthermore, tests of 5.6.51, 5.7.36 and 8.0.27 have already passed locally.